src/vip-framework/checks/checkIsolatedPoolsComptrollers.ts

```diff
@@ -138,6 +138,11 @@ const runPoolTests = async (pool: PoolMetadata, poolSupplier: string) => {
     supplyAmountScaled,
   );
   borrowAmount = borrowAmount.isZero() ? BigNumber.from(1) : borrowAmount;
+
+  // Return if Borrow is paused
+  if ((await comptroller.actionPaused(borrowMarket.address, 2)) == true) {
+    return;
+  }
```

Comment:
I would include this check in the loop where we choose the borrow market, to select a different market that is borrowable

Reply:
https://github.com/VenusProtocol/vips/pull/469/commits/351ce0aefaa1554f4b954b0853e384692151f301